packages/foreman/rubygem-public_suffix/rubygem-public_suffix.spec

```diff
@@ -1,28 +1,19 @@
-# Generated from public_suffix-3.0.3.gem by gem2rpm -*- rpm-spec -*-
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name public_suffix
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.0.3
-Release: 3%{?dist}
+Name: rubygem-%{gem_name}
+Version: 4.0.7
```

Comment:
Major version bump is due to raising the minimum Ruby version to 2.3, so it doesn't affect us.